docker.env

```diff
@@ -0,0 +1 @@
+FULLSTACK=1
```

Comment:
I think this file slipped in

Reply:
Yes, so let's update the `.gitignore` as well (since this file is created by a Makefile targets now).